src/Dapr.Actors/Resources/SR.resx

```diff
@@ -114,113 +119,154 @@
+    <comment/>
   </data>
   <data name="ErrorDeserializationFailure" xml:space="preserve">
     <value>Failed to deserialize and get remote exception  {0}</value>
+    <comment/>
   </data>
   <data name="ErrorEventInterfaceMustBeIActorEvents" xml:space="preserve">
     <value>The type '{0} is not an actor events interface. The actor event interface must only derive from '{1}'.</value>
+    <comment/>
   </data>
   <data name="ErrorExceptionSerializationFailed1" xml:space="preserve">
     <value>The exception {0} was unhandled on the service and could not be serialized for transferring to the client.</value>
+    <comment/>
   </data>
   <data name="ErrorExceptionSerializationFailed2" xml:space="preserve">
     <value>Detailed Remote Exception Information: {0}</value>
+    <comment/>
   </data>
   <data name="ErrorHeaderAlreadyExists" xml:space="preserve">
     <value>Header with name '{0}' already exists</value>
+    <comment/>
   </data>
   <data name="ErrorInterfaceNotImplemented" xml:space="preserve">
     <value>Interface id '{0}' is not implemented by object '{1}'</value>
+    <comment/>
   </data>
   <data name="ErrorInvalidAddress" xml:space="preserve">
     <value>Client is trying to connect to invalid address {0}.</value>
+    <comment/>
   </data>
   <data name="ErrorMessageHTTP404" xml:space="preserve">
     <value>The requested resource/content/path does not exist on the server.</value>
+    <comment/>
   </data>
   <data name="ErrorMethodNotImplemented" xml:space="preserve">
     <value>The object of type '{0}' does support the method '{1}'</value>
+    <comment/>
   </data>
   <data name="ErrorMethodNotSupportedInRemotingV1" xml:space="preserve">
     <value>Method '{0}' of interface '{1}' is not supported in remoting V1.</value>
+    <comment/>
   </data>
   <data name="ErrorMissingMethod" xml:space="preserve">
     <value>Method Id '{0}' for interface Id '{1}' not found in service implementation. If a new method is added to interface and client &amp; service are being upgraded at the same time, its possible that client got upgraded before the service. If a method is removed from the interface and client &amp; service are being upgraded at the same time, its possible that service got upgraded before the client. Addition or removal of methods to an interface should be performed as a phased upgrade.</value>
+    <comment/>
   </data>
   <data name="ErrorNamedActorStateNotFound" xml:space="preserve">
     <value>Actor State with name {0} was not found.</value>
+    <comment/>
   </data>
   <data name="ErrorNoServiceInterfaceFound" xml:space="preserve">
     <value>The service type '{0}' does not implement any service interfaces or one of the interfaces implemented is not a service interface. All interfaces(including its parent interface) implemented by service type must be service interface. A service interface is the one that ultimately derives from '{1}' type.</value>
+    <comment/>
```

Comment:
Can you explain or revert the changes to this file?  I'm not sure what's happening here.

Reply:
Were these changes made by visual studio?

Reply:
I will revert the changes to this file